.github/workflows/workflow.yml

```diff
@@ -22,6 +22,9 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
+      - run: npm test
+      - run: npm ci
+      - run: npm --depth 9999 update
```

Comment:
Sorry if I missed this elsewhere, but why are tests run twice?

Reply:
We test the code with the `package-lock.json` and then we test the code with the latest versions of all packages. This helps us to make sure that all the code will continue to work even if a consumer installs the latest versions of all allowed packages.

Reply:
Sounds good.